flipperdevices/flipperzero-good-faps
# Picopass: add subheader to loclass UI

# What's new

- Added "hold to reader" to locass ui

# Verification 

- Launch picopass
- Launch Loclass
- See subheader

# Checklist (For Reviewer)
